modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheDatabaseSharedManager.java

```diff
@@ -2080,24 +2080,25 @@ public PageStore getPageStore(int grpId, int partId) throws IgniteCheckedExcepti
     /**
      * @param gctx Group context.
      * @param f Consumer.
-     * @return Accumulated result for all page stores.
+     * @return Accumulated result for all page stores. Ignores all exceptions that could happen.
      */
     public long forGroupPageStores(CacheGroupContext gctx, ToLongFunction<PageStore> f) {
         int groupId = gctx.groupId();
 
         long res = 0;
 
-        try {
-            Collection<PageStore> stores = storeMgr.getStores(groupId);
+        Collection<PageStore> stores = storeMgr.getStores(groupId);
 
-            if (stores != null) {
-                for (PageStore store : stores)
+        if (stores != null) {
+            for (PageStore store : stores) {
+                try {
                     res += f.applyAsLong(store);
+                }
+                catch (Exception ignore) {
+                    // Intentionally ignore it.
```

Comment:
Do we want to log the exception at least? What kind of exceptions are we expecting here?

Reply:
We might get occasional `FileNotFoundException`, very rare. I wouldn't want to log that